.github/workflows/release-notes.yml

```
@@ -1,6 +1,12 @@
 name: Release Notes
 
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      previousVersion:
+        description: 'optional: set previous version (i.e. v0.1.0)'
```

technically this could be any kind of ref, not just a tag name

```suggestion
        description: '[Optional]: Starting ref (e.g. 8f462fa or v0.1.0)'
```